.evergreen.yml

```diff
@@ -92,6 +92,19 @@ functions:
       permissions: public-read
       display_name: storybook_assets_map_
   
+  attach-source-map:
```

Comment:
Looks like this file is not getting uploaded exactly correctly ([here](https://mciuploads.s3.amazonaws.com/parsley/parsley_ubuntu1604_small_compile_patch_445f2dabfa0ebd78a98b984fdd624c5ee8f756ca_6307c1a87742ae67c5b4a279_22_08_25_18_38_33/source_map.html))

Reply:
I updated the content_type it should work now 🤞